core/templates/dev/head/editor/feedback_tab/FeedbackTab.js

```diff
@@ -89,6 +89,47 @@ oppia.controller('FeedbackTab', [
     });
   };
 
+  $scope.showSuggestionModal = function() {
+    $modal.open({
+      templateUrl: 'modals/editorViewSuggestion',
+      backdrop: true,
+      scope: $scope,
```

Comment:
Are you referring to `scope:$scope`? I need the modal controller to be a child of FeedbackTab so I can access activeThread. Alternatively, I could pass in activeThread as a parameter when calling showSuggestionModal in ng-click, but this way seemed cleaner.